requirements.txt

```diff
@@ -5,7 +5,7 @@ boto3>=1.9.96
 apache-libcloud>=0.20.1
 click>=7.0
 yamlordereddictloader>=0.4.0
-ansible>=2.7.1,<2.7.10
```

Comment:
@14rcole 
Assuming With this current fix linchpin will work with all the versions >=2.7.1 
The requirements should be ansible>=2.7.1 since the adoption of ansible 2.8 is still low.